package.json

```diff
@@ -16,19 +16,16 @@
     "README.md",
     "dist"
   ],
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/wowserhq/wowser-math.git"
-  },
+  "repository": "wowserhq/wowser-math",
   "keywords": [
     "wowser"
   ],
-  "author": "",
+  "author": "fallenoak",
+  "contributors": [
+    "fallenoak",
+    "timkurvers"
+  ],
```

Comment:
According to the npm docs, `contributors` will source automatically from an `AUTHORS` file if one is present at the root of the package. Rather than having to modify `package.json` each time someone new contributes, let's stick to that default.

https://docs.npmjs.com/files/package.json#default-values

> If there is an AUTHORS file in the root of your package, npm will treat each line as a Name <email> (url) format, where email and url are optional. Lines which start with a # or are blank, will be ignored.

Reply:
Ah neat, I didn't see that at the bottom of the documentation. Should I go ahead and add that here?

Reply:
👍 Yeah, let's add an `AUTHORS` file with our individual names in it.